src/openrct2/network/NetworkBase.cpp

```diff
@@ -4248,8 +4239,8 @@ NetworkServerState_t network_get_server_state()
 {
     return NetworkServerState_t{};
 }
-json_t* network_get_server_info_as_json()
+json_t network_get_server_info_as_json()
 {
-    return nullptr;
+    return json_t();
```

Comment:
```suggestion
    return {};
```

PS: I kind wonder how useful this function really is